resources.tf

```diff
@@ -150,42 +150,171 @@ resource "aws_iam_role_policy_attachment" "ec2_for_ssm_attachment" {
 }
 
 #        Create and attach custom policy
-#        TODO Lock down analytical dataset EMR instance profile DW-3618
 data "aws_iam_policy_document" "analytical_dataset_write_s3" {
+
   statement {
     effect = "Allow"
 
     actions = [
-      "acm:ExportCertificate",
-      "cloudwatch:*",
+      "ec2:AuthorizeSecurityGroup*",
```

Comment:
These all need splitting out into their own service-specific or resource-specific statements so we can control access to specific resources. I'd be prone to just trying to see what happens when this catch-all statement is removed because it's making a lot of the more specific rules underneath it null and void.

Reply:
Amended the code by mentioning required services